plugins/language_rivet.lua

```diff
@@ -1,7 +1,7 @@
 -- mod-version:3
 
 -- Syntax highlighting for the Rivet programming language.
--- by StunxFS :)
+-- By StunxFS.
```

Comment:
:(

Reply:
We need the happy face, right? I forgot to put it in x,D. I'll put it back in another PR :).